vowpalwabbit/cats_tree.cc

```diff
@@ -92,15 +92,15 @@ void min_depth_binary_tree::build_tree(uint32_t num_nodes, uint32_t bandwidth)
       if (bandwidth)
       {
         right_only = (id == (_num_leaf_nodes / (2 * bandwidth) - 1));
-        left_only = (id == (_num_leaf_nodes / (bandwidth)-2));
+        left_only = (id == (_num_leaf_nodes / (bandwidth) - 2));
```

Comment:
why isnt there symmetry between right_only and left_only (as in same formula?)

Reply:
it's derived from the [paper](https://arxiv.org/pdf/2006.06040.pdf) algorithm 9 page 27